### ISSUE TITLE
Advice for moving from clue/metrics to clue OTEL usage

### ISSUE THREAD

**Issue by ryanluu12345** [goadesign/clue#377]
Previously, we were using `clue/metrics` which allowed us to send metrics to Prometheus directly. It seems like the `metrics.HTTP` is no longer available in its previous form. It seems that the new suggested method is to use OTEL, which I understand needs an OTEL collector.

By any chance, does the new methodology support direct Prometheus metrics? If not, do you have any advice for migrating to the new methodology?

**Comment by raphael** [comment]
Hello, migrating to OpenTelemetry opens up the [entire ecosystem](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/b16a03f946b7e26c94bf2aae11b429cd8931013f/exporter) to clue users but also means that the simple direct integration with Prometheus isn't practical anymore. Migrating to clue 1.x requires running an OpenTelemetry collector that exposes the Prometheus compatible endpoint.

Here is the Quick start guide for running an OpenTelemetry collector: https://opentelemetry.io/docs/collector/quick-start/ - basically it consists of running a docker container that is configured via a [YAML file](https://opentelemetry.io/docs/collector/configuration/). 

And here is the [README](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/b16a03f946b7e26c94bf2aae11b429cd8931013f/exporter/prometheusexporter) for the Prometheus exporter. 

So TL; DR:
* Create a YAML file with content similar to:
```yaml
receivers:
  otlp:
    protocols:
      grpc:
exporters:
  prometheus:
    endpoint: "0.0.0.0:8889"
processors:
  batch:
service:
  pipelines:
    metrics:
      receivers: [otlp]
      processors: [batch]
      exporters: [prometheus]
```
* Run the OpenTelemetry collector docker image with the config above:
```bash
docker run \
  -p 127.0.0.1:4317:4317 \
  -p 127.0.0.1:55679:55679 \
  -v $(pwd)/config.yaml:/etc/otel/config.yaml \
  otel/opentelemetry-collector \
  2>&1 | tee collector-output.txt # Optionally tee output for easier search later
```

Hope this helps!
